app/config/message.js

```diff
@@ -2,23 +2,23 @@ const Joi = require('joi')
 const { PRODUCTION } = require('../constants/environments')
 
 const schema = Joi.object({
-  eventsQueue: {
+  crmQueue: {
```

Comment:
Previously used `eventsQueue` and `eventsSubscription` when `events` was in the name of the Service Bus topic, but based on Abi's feedback from [a previous PR](https://github.com/DEFRA/ffc-sfd-crm/pull/4#discussion_r1687719777) to change the topic name (`ffc-sfd-crm-events` -> `ffc-sfd-crm-messages`), I've opted to change this to `crmQueue` and `crmSubscription` for the time being so it's more consistent. Can review in future and update if needed.